spec/facility_list_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "./lib/facility.rb"
+require "./lib/facility_list.rb"
+require "./lib/dmv_data_service.rb"
+
+describe FacilityList do
+    it "exists" do
+        factory = FacilityList.new
+
+        expect(factory).to be_a FacilityList
+    end
+
+    it "creates facilities with CO API data" do
+        list = FacilityList.new
+        co_facilities_data = DmvDataService.new.co_dmv_office_locations
+        list.parse_facilities(co_facilities_data)
+
+        expect(list.facilities.first).to be_a(Facility)
+        expect(list.facilities.first.name).not_to be_nil
+        expect(list.facilities.first.address).not_to be_nil
+        expect(list.facilities.first.phone).not_to be_nil
+        expect(list.facilities.first.services).not_to be_nil
+        expect(list.facilities.first.hours).not_to be_nil
```

Comment:
Yup, this is good. This is really all we can test with an Api where we don't know the data.